.ci/install_runtime.sh

```diff
@@ -53,22 +53,22 @@ echo "Enabling global logging for runtime in file ${runtime_config_path}"
 sudo sed -i -e 's/^#\(\[runtime\]\|global_log_path =\)/\1/g' "${runtime_config_path}"
 
 echo "Add runtime as a new/default Docker runtime. Docker version \"$(docker --version)\" could change according to Semaphore CI updates."
-docker_options="-D --add-runtime cc30=/usr/local/bin/cc-runtime --default-runtime=cc30"
+docker_options="--add-runtime cc30=/usr/local/bin/cc-runtime --default-runtime=cc30"
```

Comment:
What issues are we seeing with `-D`? This is a useful option so we don't really want to have to remove it imho.

Reply:
I don't know why I was not able to restart the docker service in the past when this was enabled but apparently I have this working now. Nevermind...